homeassistant/components/cast/media_player.py

```diff
@@ -931,12 +931,12 @@ def media_stop(self):
     def media_previous_track(self):
         """Send previous track command."""
         media_controller = self._media_controller()
-        media_controller.rewind()
+        media_controller.queue_next()
```

Comment:
We could consider seeking to beginning of track if more than a few seconds has been played.
But maybe that's something that should be handled in the frontend instead of by each integration?

Reply:
I think that it's something that should be handled by each integration, preferably we even just forward the command and let the service/device itself figure it out.